migrations/1_create_users_projects_files.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+module.exports = {
+  up: function(migration, DataTypes, done) {
+    migration.createTable(
+      'Users', {
+        name: DataTypes.STRING
+      });
+    migration.createTable(
+      'Projects', {
+        title: DataTypes.STRING,
+        user_name: DataTypes.STRING
+      });
+    migration.createTable(
+      'Files', {
+        path: DataTypes.STRING,
+        type: DataTypes.STRING,
```

Comment:
I don't know if we want to store the content type in the db. I think we should determine it based on the file extension on-the-fly when we serve it